File: 1546.c
#include <stdio.h>

int main() {
    int N, K, i, feedback;

    scanf("%d", &N);

    for(i = 0; i < N; i++) {
        scanf("%d", &K);
        for(int j = 0; j < K; j++) {
            scanf("%d", &feedback);
            if(feedback == 1)
                printf("Rolien\n");
            else if(feedback == 2)
                printf("Naej\n");
            else if(feedback == 3)
                printf("Elehcim\n");
            else
                printf("Odranoel\n");
        }
    }

    return 0;
}

File: 1546.py
N = int(input())
for i in range(N):
    K = int(input())
    for j in range(K):
        feedback = int(input())
        if feedback == 1:
            print("Rolien")
        elif feedback == 2:
            print("Naej")
        elif feedback == 3:
            print("Elehcim")
        else:
            print("Odranoel")
